### ISSUE TITLE
how to use your deploy.prototoxt to 2 classification

### ISSUE THREAD

**Issue by gittigxuy** [shicai/DenseNet-Caffe#16]
In my opinion, in order to 2 classification,I should add the probability which type is softmax to achieve it.but I find that in your DenseNet_121.prototxt file,I could not find the softmax layer.So in order to achieve the 2 classification,what should I do to output the probability,and what should I do to achieve 2 classification

**Comment by shicai** [comment]
you should finetune this model for your own binary classification tasks.

**Comment by gittigxuy** [comment]
那这个神经网络都没有prob（概率层），怎么进行分类预测呢？mnist十分类基于alexnet，它的deploy.prototxt是有prob层的，你这个都没有Prob层，我加上prob层会莫名其妙的报错，加不进去这个层，请问一下怎么解决？

**Comment by shicai** [comment]
1. 请尽量先熟悉caffe（很重要）
2. 可以加prob层，如果你的有错误的话，你可以贴出你怎么加的，出现什么错？
